instance.tf

```diff
@@ -11,6 +11,22 @@ resource "aws_security_group" "allow_ssh" {
     protocol    = "tcp"
     cidr_blocks = ["70.95.138.224/32"]
 
+  }
+  ingress {
+    description = "http to VPC"
+    from_port   = 80
+    to_port     = 80
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+
+  }
+  ingress {
+    description = "https to VPC"
+    from_port   = 443
+    to_port     = 443
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+
```

Comment:
We are not using port 443 yet. This means there is no need to have this rule.
It's a good idea to only keep code that is used right here and right now.